cli/src/commands/state/get.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2023 -  Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::c_title;
+use crate::cli_env::CliEnv;
+use crate::console::c_println;
+use crate::ui::console::StyledTable;
+use crate::ui::watcher::Watch;
+
+use crate::commands::state::util::{as_json, get_current_state, pretty_print_json_object};
+use anyhow::Result;
+use cling::prelude::*;
+use comfy_table::{Cell, Table};
+
+#[derive(Run, Parser, Collect, Clone)]
+#[cling(run = "run_get")]
+pub struct Get {
+    /// Don't try to convert the values to a UTF8 string
+    #[clap(long, alias = "bin")]
+    binary: bool,
+
+    /// Don't try to convert the values to a UTF8 string
+    #[clap(long, alias = "s")]
+    silent: bool,
```

Comment:
Silent might be confusing here since it's not really "silent", would`--plain` or `--json` be a better option? See https://clig.dev/#output for suggestions.